pkg/netpol/connlist/conns_formatter_md.go

```diff
@@ -8,33 +8,59 @@ package connlist
 
 import (
 	"fmt"
-	"sort"
 	"strings"
 )
 
 // formatMD: implements the connsFormatter interface for md output format
 type formatMD struct {
+	ipMaps ipMaps
 }
 
-// formats the md output header
+// getMDHeader formats the md output header
 func getMDHeader() string {
 	return "| src | dst | conn |\n|-----|-----|------|"
 }
 
-// formats a connection line for md output
+// getMDLine formats a connection line for md output
 func getMDLine(c singleConnFields) string {
 	return fmt.Sprintf("| %s | %s | %s |", c.Src, c.Dst, c.ConnString)
 }
 
-// returns a md string form of connections from list of Peer2PeerConnection objects
-// this format is not supported with exposure analysis; exposureConns is not used;
+// writeOutput returns a md string form of connections from list of Peer2PeerConnection objects,
+// and exposure analysis results from list ExposedPeer if exists
 func (md *formatMD) writeOutput(conns []Peer2PeerConnection, exposureConns []ExposedPeer) (string, error) {
-	mdLines := make([]string, len(conns))
-	for index := range conns {
-		mdLines[index] = getMDLine(formSingleP2PConn(conns[index]))
-	}
-	sort.Strings(mdLines)
+	exposureFlag := len(exposureConns) > 0
```

Comment:
could it be that when running with `--exposure` there is not exposure results, but then the output does not add the header? isn't it a bit misleading output?

Reply:
i can instead of checking the exposure result's length , to check the exposure analysis flag (on with --exposure)
and if the result empty, to print header with empty section

Reply:
ok, sounds good

Reply:
done